test/functional/interface_rpc.py

```diff
@@ -67,10 +80,23 @@ def test_http_status_codes(self):
         expect_http_status(404, -32601, self.nodes[0].invalidmethod)
         expect_http_status(500, -8, self.nodes[0].getblockhash, 42)
 
+    def test_work_queue_exceeded(self):
+        self.log.info("Test work queue exceeded")
+        self.restart_node(0, ['-rpcworkqueue=1'])
```

Comment:
This would make CI testing more reliable:
```suggestion
        self.restart_node(0, ["-rpcworkqueue=1", "-rpcthreads=1"])
```